demo/web/src/main/notebooks/07 Data Replay.md

```diff
@@ -0,0 +1,72 @@
+# Data Replay
+
+Deephaven excels at handling live data. Integrating historical data into real-time analysis is common in a multitude of fields including machine learning, validation, modeling, simulation, and forecasting.
+
+Here we will show how to take historical data and play it back as real-time data based on timestamps in a table. This example could be easily extended towards a variety of real-world applications.
+
+To start, let's make a sample table containing random numbers generated at certain historical time stamps.
```

Comment:
```suggestion
To start, let's make a sample table containing random numbers generated at certain historical timestamps.
```

Reply:
Standardize timestamp as one word - it's evidently the British way :) but we mostly use it as one word in the docs